README.md

```diff
@@ -1,45 +1,122 @@
-# Website
+# Temporal documentation repository
 
-This website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
+This repo contains Temporal's open source documentation.
 
-### Prerequisites
+You can view the published documentation at https://docs.temporal.io.
 
-Install [YARN](https://classic.yarnpkg.com/en/docs/install/).
+This repo utilizes the [Docusaurus 2](https://v2.docusaurus.io/) framework, a modern static website generator, which allows you to build and view the documentation locally as you contribute.
 
-### Installation
+## Contribution guidelines
+
+Contributions from the community are highly encouraged.
+
+To contribute please follow these steps.
+
+### Step 1: Fork and clone
+
+To ensure that each change is properly peer reviewed, we request that you follow standard open source contribution processes.
+
+This means that you should start by forking the project into your own GitHub account.
```

Comment:
```suggestion
Start by forking the project into your own GitHub account and then cloning that fork onto your local machine.
```

Remove other sentence.

Reply:
These are two separate steps. Since it is followed by a command, one could be confused and assume that the command does both.

In this case it is better to have two separate sentences to address each step in the process.

I will add a screen shot after the first sentence -